### DISCUSSION TITLE
PWSH support for windows

### DISCUSSION THREAD

**Discussion by dartains** [eclipse-theia/theia#14597] [Ideas]
I tried to configure powershell, but it always open CMD.
There are other discussions with the same topic, but no comments/answers.
![image](https://github.com/user-attachments/assets/9a222e07-5b4c-402e-ac08-7533af558b0a)

**Comment by msujew** [comment]
Hey @dartains, please see https://github.com/eclipse-theia/theia/discussions/13792#discussioncomment-11526573 for a solution.
